system/lang/en/menu.txt

```diff
@@ -657,155 +657,155 @@ loadingLocations = "Loading locations"
+rumor_24 = "The shield Wall looks like a piece of wall but it is much harder. The leader of the dark priests has it."
+rumor_25 = "Every year the tournament takes place in the arena, exactly at 7.03. It's hard to win, depends on who comes."
+rumor_26 = "The moonstone has something to do with the moon well, but what?"
+rumor_27 = "Here and there you can find a strange obelisk. No one knows who built them or even when."
+rumor_d_0 = "Don't tell anyone, apparently the innkeeper pisses in the beer!"
+rumor_d_1 = "This beer is so watered down!"
 rumor_d_2 = "The beer here is piss, the best is in $rcitynhere$!"
-rumor_d_3 = "The contest in drinking takes place at 21.09 in some tavern. Can get fucking drunk for free!"
+rumor_d_3 = "The drinking contest takes place at 21.09 in some tavern. You can get fucking drunk for free!"
 rumor_d_4 = "This spirit is weak, I would like to have something stronger..."
 rumor_d_5 = "I ain't (hic) fucking drunk!"
-rumor_d_6 = "There are no ugly women, only sometimes the lack of wine..."
+rumor_d_6 = "There are no ugly women, only a lack of wine sometimes..."
 
 // dialogi 1
 mayorQFailed0 = "I can't rely on you..."
-mayorQFailed1 = "I will not give you any important task! Go collect the carrots!"
-mayorQFailed2 = "I needs someone I can rely on, on you I cannot."
+mayorQFailed1 = "I won't give you any important task! Go collect carrots instead!"
+mayorQFailed2 = "I needs someone I can rely on, not you."
 questAlreadyGiven0 = "I already assigned you a task."
 questAlreadyGiven1 = "First, take care of what I commissioned you."
-mayorNoQ0 = "Currently I have no job for you. Come back in some time."
-mayorNoQ1 = "We currently do not need your help, maybe in the future."
+mayorNoQ0 = "Currently, I have no job for you. Come back in some time."
+mayorNoQ1 = "We don't need your help currently, maybe in the future."
 captainQFailed0 = "You are not suitable for such tasks."
-captainQFailed1 = "I need people who may come in handy, you have failed already."
-captainNoQ0 = "Currently, everything is quiet in the area. We'll see what will happen in some time."
-captainNoQ1 = "I don't have any job for you in this moment."
+captainQFailed1 = "I need people who may come in handy, you have already failed."
+captainNoQ0 = "Currently, everything is quiet in this area. We'll see what will happen after some time."
+captainNoQ1 = "I don't have any job for you this moment."
 locationDiscovered0 = "Apparently at %s%s of here is %s from what I heard."
 locationDiscovered1 = "People say that on %s%s from here there is %s."
 allDiscovered0 = "I wanted to tell you about some unknown locations but it seems that you already know everything."
 allDiscovered1 = "So many locations... And you were everywhere!"
-campDiscovered0 = "Apparently at %s%s there is camp of %s. I hope that they will not attack a village."
+campDiscovered0 = "Apparently there is camp of %s at %s%s. I hope that they won't attack a village."
```

Comment:
Currently this won't work as string format argument will be in incorrect order. It will show as: Apparently there is camp of [direction] at [location][enemy name].